packages/abstract-sdk/src/apis/bank.rs

```diff
@@ -148,6 +150,20 @@ impl<'a, T: TransferInterface> Bank<'a, T> {
         let recipient = &env.contract.address;
         self.transfer(funds, recipient)
     }
+
+    /// Move cw20 assets from the Account to a recipient with the possibility using the cw20 send/receive hook
```

Comment:
Could you add a warning there that native coins won't be supported. in **bold** to make it scary haha

Reply:
This is resolved in https://github.com/AbstractSDK/contracts/pull/370/commits/7d593f3478bb588e663996a9200e390805311841